source/gameengine/Converter/KX_ConvertActuators.cpp

```diff
@@ -977,7 +977,7 @@ void BL_ConvertActuators(const char* maggiename,
 			}
 
 			tmp = new SCA_2DFilterActuator(gameobj, filtermode,  _2dfilter->flag,
-			                               _2dfilter->float_arg, _2dfilter->int_arg,
+			                               _2dfilter->float_arg, _2dfilter->int_arg, _2dfilter->mipmap,
```

Comment:
the nsca_2dfilteractuator constructor is:
SCA_2DFilterActuator(
	        class SCA_IObject* gameobj,
	        int type,
	        short flag,
	        float float_arg,
	        int int_arg,
	        int mipmap,
	        RAS_IRasterizer* rasterizer,
			RAS_2DFilterManager *filterManager,
	        SCA_IScene* scene);